### ISSUE TITLE
new improved typings not published

### ISSUE THREAD

**Issue by loreanvictor** [atomrc/callbag-debounce#11]
It appears that the published version of the package [does not have the improved typings](https://unpkg.com/browse/callbag-debounce@2.1.2/dist/debounce.d.ts). This is the code on master:

```ts
export function debounce<I>(wait: number): (source: Source<I>) => Source<I> {
  // ...
}
```

while this is the last published version:

```ts
export declare function debounce(wait: number): any;
```

Are there any plans to publish again so the improved typings are also published? It appears that they were merged 8 months ago.

**Comment by atomrc** [comment]
The new types are published under `v2.1.3` 👍

**Comment by loreanvictor** [comment]
Thanks!
